doc/api/api_changes.rst

```diff
@@ -11,6 +11,19 @@ sources of the changes you are experiencing.
 For new features that were added to matplotlib, please see
 :ref:`whats-new`.
 
+Changes in 2.1.0
+================
+
+Code changes
```

Comment:
Can you put these is a file in `api_changes/*.rst`.  This helps to prevent the sort of conflicts this branch has.